samples/asgardeo-express-app/index.js

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Can we add a web interface similar to the node sample app?

Reply:
I'll add both